components/tikv_kv/src/lib.rs

```diff
@@ -1,8 +1,8 @@
 // Copyright 2021 TiKV Project Authors. Licensed under Apache-2.0.
 
-//! There are multiple [`Engine`](kv::Engine) implementations, [`RaftKv`](crate::server::raftkv::RaftKv)
-//! is used by the [`Server`](crate::server::Server). The [`BTreeEngine`](kv::BTreeEngine) and
-//! [`RocksEngine`](RocksEngine) are used for testing only.
+//! There are multiple [`Storage`](kv::Storage) implementations, [`RaftKv`](crate::server::raftkv::RaftKv)
```

Comment:
let's rename `RaftKv` to `RaftKvStorage` ?

Reply:
And maybe rename the component from `tikv_kv` to `storage`, and rename `src/storage/kv` to `src/storage_handler/storage` ?

Reply:
There are a lot of changes and `tikv_kv` looks OK to me.